### PR TITLE
Added a launch file to display a urdf.xacro model

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

File: cob_hardware_config/CMakeLists.txt [hunk 1]
```diff
@@ -25,3 +25,7 @@ install(DIRECTORY robots test
 install(FILES upload_robot.launch
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
+
+install(FILES xacrodisplay.launch
```
Comment (fmessmer) [inline cob_hardware_config/CMakeLists.txt]: better name `display_robot.launch`?
(you could also combine the two `FILES` install tags...

File: cob_hardware_config/xacrodisplay.launch [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
```
Comment (fmessmer) [inline cob_hardware_config/xacrodisplay.launch]: `upload_robot.launch` uses tabs for indentation

File: cob_hardware_config/xacrodisplay.launch [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+<launch>
+  <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+  <arg name="gui" default="false" />
+  <arg name="rvizconfig" default="$(find urdf_tutorial)/urdf.rviz" />
```
Comment (fmessmer) [inline cob_hardware_config/xacrodisplay.launch]: these introduces an `exec_depend>` to `urdf_tutorial`...
how about copying that file...or simply use `cob_hardware_config/robots/$(arg robot)/$(arg robot).rviz` (should be fine enough having a RobotModel loaded to the displays...

File: cob_hardware_config/xacrodisplay.launch [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+  <arg name="gui" default="false" />
```
Comment (fmessmer) [inline cob_hardware_config/xacrodisplay.launch]: default `true`?

**Review by fmessmer — COMMENTED** [review]
some comments

File: cob_hardware_config/xacrodisplay.launch (hunk 3 above)
Comment (NadiaHG) [reply cob_hardware_config/xacrodisplay.launch]: the `cob_hardware_config/robots/$(arg robot)/$(arg robot).rviz` includes the cameras, the sensors data, map... all of this is not needed for the purpose of the file. I propose copy the file to `cob_hardware_config/robots/common/display_robot.rviz`, is this ok?

File: cob_hardware_config/xacrodisplay.launch (hunk 4 above)
Comment (NadiaHG) [reply cob_hardware_config/xacrodisplay.launch]: I prefer false, I only use the gui modus to check the limits...

File: cob_hardware_config/display_robot.launch [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+
+	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+	<arg name="gui" default="false" />
+	<arg name="rvizconfig" default="$(find cob_hardware_config)/robots/config/display_robot.rviz" />
```
Comment (fmessmer) [inline cob_hardware_config/display_robot.launch]: the path of the file is `robots/common`!

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: cob_hardware_config/xacrodisplay.launch (hunk 3 above)
Comment (floweisshardt) [reply cob_hardware_config/xacrodisplay.launch]: I vote for using the existing rviz configs. it does not hurt if there are laser scanners and cameras defined in the config. if we introduce a new rviz config thats one more file we have to maintain without gaining mayor benefits.

**Review by floweisshardt — CHANGES_REQUESTED** [review]
I vote for using the existing rviz configs. it does not hurt if there are laser scanners and cameras defined in the config. if we introduce a new rviz config thats one more file we have to maintain without gaining mayor benefits.

**Review by fmessmer — APPROVED** [review]

**Comment by fmessmer** [comment]
I leave it to @ipa-fmw and @ipa-nhg to decide about the config and then merge this PR...I'm ok with both options...

**Comment by floweisshardt** [comment]
@ipa-nhg: let's fight :-)

**Review by floweisshardt — APPROVED** [review]

**Pull request merged**